sphinxcontrib/katex-server.js

```diff
@@ -0,0 +1,140 @@
+const net = require("net");
+const process = require("process");
+
+let value = null;
+let katex_options = {};
+let socket = null;
+let port = null;
+// this is the downloaded katex javascript library.
+// need to keep it updated
+let katex_path = "./katex";
```

Comment:
I guess this refers to the `katex.js` file, correct?
Maybe add it to the comment to make it more clear:
```js
// path to the downloaded katex javascript library `katex.js`.
```

Reply:
I will add this.